app/controllers/declaration/SummaryController.scala

```diff
@@ -70,7 +70,7 @@ class SummaryController @Inject()(
             BadRequest(summaryPage(Mode.Normal, SupplementaryDeclarationData(request.cacheModel), formWithErrors))
         ),
         legalDeclaration => {
-          submissionService.submit(request.cacheModel, legalDeclaration).map {
+          submissionService.submit(request.authenticatedRequest.user.eori, request.cacheModel, legalDeclaration).map {
```

Comment:
I'm not sure but we should have `eori` field in `JourneyRequest` so you don't need to go through `authenticatedRequest.user.eori`

Reply:
It wasn't on the request but it is now 👍